src/mlpack/core/optimizers/lbfgs/test_functions.cpp

```diff
@@ -89,7 +89,7 @@ double WoodFunction::Evaluate(const arma::mat& coordinates)
                      /* f3(x) */ 90 * std::pow(x4 - std::pow(x3, 2), 2) +
                      /* f4(x) */ std::pow(1 - x3, 2) +
                      /* f5(x) */ 10 * std::pow(x2 + x4 - 2, 2) +
-                     /* f6(x) */ (1 / 10) * std::pow(x2 - x4, 2);
+                     /* f6(x) */ (1.0 / 10.0) * std::pow(x2 - x4, 2);
```

Comment:
Looks like this broke the test.

Reply:
I was surprised, it turns out that the same casting issue existed in the gradient, but I missed it and so did the static analyzer (probably because it was on a separate line)?  Anyway a fun little bug to figure out. :)